strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/ConfigurationManagementServiceImpl.java

```diff
@@ -618,4 +624,57 @@ private void modifyInLock(final Consumer<MutableConfiguration> operation,
         }
     }
 
+    @Override
+    public boolean setMaxUploadFileSize(String maxUploadSizeString) throws SizeFormatValidationException
+    {
+        if (!(this.multipartConfigElement instanceof UpdatableMultipartConfigElement))
+        {
+            return false;
+        }
+
+        maxUploadSizeString.replaceAll("\\s+","");
```

Comment:
Result of `replaceAll` is ignored. Please, assign it to a variable.
```suggestion
        maxUploadSizeString = maxUploadSizeString.replaceAll("\\s+", "");
```